Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -354,13 +524,53 @@ bool isFunctionJson(string fullName)
             }
         }
 
-        private async Task<string> DeployZipLocally(ZipDeploymentInfo zipDeploymentInfo, ITracer tracer)
+        private async Task ArtifactFetch(IRepository repository, DeploymentInfoBase deploymentInfo, string targetBranch, ILogger logger, ITracer tracer)
```

Comment:
OneDeployFetch would be a better name here.

Reply:
Also, you want to add a comment that says that this will be used for OneDeploy artifacts that are not zip files.